src/lib/stitching/vortex/stitching.ts

```diff
@@ -318,14 +318,27 @@ export const vortexStitchingEnvironment = (localSchema: GraphQLSchema) => ({
               ... on AnalyticsArtist {
                 entityId
               }
+              ... on AnalyticsViewingRoom {
+                entityId
+              }
             }
           }
         `,
         resolve: (parent, _args, context, info) => {
           const removeVortexPrefix = (name) => name.replace("Analytics", "")
           const typename = parent.rankedEntity.__typename
-          const fieldName = removeVortexPrefix(typename).toLowerCase()
+          // TO CLEAN BEFORE MERGE ---------------
+          // This fieldName line below is a point of failure here since
+          // the type name comes through as "AnalyticsViewingRoom"
+          // and the toLowerCase changes the case to "viewingroom"
+          // we need to match the gravity schema case of "viewingRoom"
+          const typenameWithoutPrefix = removeVortexPrefix(typename)
+          const fieldName =
+            typenameWithoutPrefix.charAt(0).toLowerCase() +
+            typenameWithoutPrefix.slice(1)
+          // -------------------------------------------------
```

Comment:
Is this comment outdated? I think as-written it should work - going from `AnalyticsViewingRoom` -> `VieiwngRoom` -> `viewingRoom`, right?

Reply:
Correct! old comment here, this should be working with the new addition of likes 335 - 338 - will remove